installer/manifests/keptn/charts/control-plane/templates/ingress.yaml

```diff
@@ -1,3 +1,47 @@
+{{- if lt .Capabilities.KubeVersion.Minor "19" }}
```

Comment:
Why did you create a complete separate ingress manifest here, instead of putting the if/elses in the manifest below?

Reply:
It's a matter of choise... If I will add if/else to the existing manifest, it will require changes on two places (2x if/else) -> `apiVersion` and `spec`. The current change requires only one time if/else, but the manifest is longer. It's actually upon us, so if you like, I can easily change it. It actually does not matter that much.

Reply:
I'm always in favour of reducing lines of code but I respect your decision of course if you want to leave it as it is :)

Reply:
fixed, re-review? :)